src/Sylius/Bundle/InstallerBundle/Command/SetupCommand.php

```diff
@@ -117,198 +108,73 @@ protected function setupAdministratorUser(InputInterface $input, OutputInterface
 
             $valid = true;
 
-            foreach ($currencies as $code) {
-                if (0 !== count($errors = $this->validate(trim($code), [new Currency()]))) {
-                    $valid = false;
-                }
-
-                $this->writeErrors($output, $errors);
-            }
-        } while (!$valid);
-
-        foreach ($currencies as $key => $code) {
-            $code = trim($code);
-
-            $name = Intl::getCurrencyBundle()->getCurrencyName($code);
-            $output->writeln(sprintf('Adding <info>%s</info>.', $name));
-
-            if (null !== $existingCurrency = $currencyRepository->findOneByCode($code)) {
-                $this->currencies[] = $existingCurrency;
-
-                continue;
+            if (0 !== count($errors = $this->validate(trim($code), [new Currency()]))) {
+                $valid = false;
             }
 
-            $currency = $currencyFactory->createNew();
-            $currency->setCode($code);
-            $currency->setExchangeRate(1);
-            $this->currencies[] = $currency;
-
-            $currencyManager->persist($currency);
-        }
-
-        $currencyManager->flush();
-    }
-
-    /**
-     * @param InputInterface  $input
-     * @param OutputInterface $output
-     */
-    protected function setupCountries(InputInterface $input, OutputInterface $output)
-    {
-        $countryRepository = $this->get('sylius.repository.country');
-        $countryManager = $this->get('sylius.manager.country');
-        $countryFactory = $this->get('sylius.factory.country');
-
-        do {
-            $countries = $this->getCountriesCodes($input, $output);
-
-            $valid = true;
-
-            foreach ($countries as $code) {
-                if (0 !== count($errors = $this->validate(trim($code), [new Country()]))) {
-                    $valid = false;
-                }
 
                 $this->writeErrors($output, $errors);
-            }
         } while (!$valid);
 
-        foreach ($countries as $key => $code) {
-            $code = trim($code);
-            $name = Intl::getRegionBundle()->getCountryName($code);
+        $code = trim($code);
+        $name = Intl::getCurrencyBundle()->getCurrencyName($code);
+        $output->writeln(sprintf('Adding <info>%s</info>', $name));
 
-            $output->writeln(sprintf('Adding <info>%s</info>.', $name));
+        if (null !== $existingCurrency = $currencyRepository->findOneByCode($code)) {
+            $this->currency = $existingCurrency;
 
-            if (null !== $countryRepository->findOneByCode($code)) {
-                continue;
-            }
+            return;
+        }
 
-            $country = $countryFactory->createNew();
-            $country->setCode($code);
+        $currency = $currencyFactory->createNew();
 
-            $countryManager->persist($country);
-        }
+        $currency->setExchangeRate(1);
+        $currency->setBase(true);
```

Comment:
When I run this command twice with different currencies I will have two base currencies in the database... We should validate that (in a separate PR).